letsdrill.rb

```diff
@@ -1,12 +1,20 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  @integer=integer
+    case
+	  when integer.between?(90,100) then return "A"
```

Comment:
check out the styling suggestions on case statements [here](https://github.com/bbatsov/ruby-style-guide).  The biggest thing would be do `case integer`, instead of repeating `integer` on every line.